weaviate/collections/data.py

```diff
@@ -470,9 +480,7 @@ def update(
 
         self._update(weaviate_obj, uuid=uuid)
 
-    def reference_add(
-        self, from_uuid: UUID, from_property: str, to: Union[UUID, WeaviateReference]
-    ) -> None:
+    def reference_add(self, from_uuid: UUID, from_property: str, to: WeaviateReference) -> None:
```

Comment:
Should we extend this type to be `Union[WeaviateReference, List[UUID]]` and so allow the the creation of one-to-many `(1-N)` references in a single method call? I see you did this with `reference_replace`

This would mean that `N` synchronous requests would be sent within the method, being therefore highly blocking, but the async connection might help here to schedule them concurrently

Reply:
reference_replace can sends lists as one request, that's why I added it there.

If users want to do multiple adds they can just use reference_add_many and/or batch :)